awx/main/tests/functional/test_inventory_source_injectors.py

```diff
@@ -192,10 +193,16 @@ def create_reference_data(source_dir, env, content):
 
 
 @pytest.mark.django_db
-@pytest.mark.parametrize('this_kind', CLOUD_PROVIDERS)
-def test_inventory_update_injected_content(this_kind, inventory, fake_credential_factory, mock_me):
+@pytest('this_kind', discover_available_cloud_provider_plugin_names)
+def test_inventory_update_injected_content(this_kind, inventory, fake_credential_factory, self):
```

Comment:
Deleting parametrize means that `this_kind` would need to be injected from a fixture, but since it doesn't exist, and it's not supposed to be produced early (unless a fixture is declared), I'd just drop it. Additionally, the `mock_me` fixture injection has been lost, and a fixture called `self` probably does not exist.
```suggestion
def test_inventory_update_injected_content(inventory, fake_credential_factory, mock_me):
```